src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -276,7 +276,7 @@ const ActivityPageContent = styled(PageContent)`
 `;
 
 const Sidebar = styled('div')`
-  width: 40%;
+  width: 400px;
```

Comment:
The mocks give this a fixed with, which I think makes sense. But maybe not 🤷‍♂️

Reply:
I would like to find a way to standardize the 66/33 or 60/40 layout as we use in several places (discover, alerts, issues) and we have a few different implementations of it.

Reply:
Yeah we have it in quite a few places. Hopefully we can start standardizing on some components for things like that